cli/create.go

```diff
@@ -54,6 +54,9 @@ func (cc *CreateCommand) runCreate(args []string) error {
 	}
 
 	config.Image = args[0]
+	if !strings.Contains(config.Image, ":") {
+		config.Image += ":latest"
```

Comment:
I think Image here should be passed by name:tag, and we should not append tag for Image if it is not passed. Just proposal.

Reply:
I added image not empty validate, and I think, if not passed image name, it should not execute here.